src/net_processing.cpp

```diff
@@ -1746,8 +1748,12 @@ void PeerManagerImpl::RelayAddress(NodeId originator,
     // Relay to a limited number of other nodes
     // Use deterministic randomness to send to the same nodes for 24 hours
     // at a time so the m_addr_knowns of the chosen nodes prevent repeats
-    const uint64_t hashAddr{addr.GetHash()};
-    const CSipHasher hasher{m_connman.GetDeterministicRandomizer(RANDOMIZER_ID_ADDRESS_RELAY).Write(hashAddr).Write((GetTime() + hashAddr) / (24 * 60 * 60))};
+    uint64_t hashAddr = addr.GetHash();
```

Comment:
why is this changed?

Reply:
left that line unchanged in the rebase at https://github.com/jonatack/bitcoin/commit/77ccb7fce15e340080f14c7626cf3dc63fcdee88